lib/fog/dns/openstack/v2/requests/create_zone_transfer_accept.rb

```diff
@@ -3,13 +3,14 @@ module DNS
     class OpenStack
       class V2
         class Real
-          def create_zone_transfer_accept(key, zone_transfer_request_id)
+          def create_zone_transfer_accept(key, zone_transfer_request_id,target_project_id=nil)
             data = {
               :key => key,
               :zone_transfer_request_id => zone_transfer_request_id
             }
 
             request(
+              :headers => target_project_id.nil? ? nil : {"x-auth-sudo-project-id" => target_project_id},
```

Comment:
Please use `Fog::DNS::OpenStack::V2.setup_headers` for this. It takes care of adding the header when given the `project_id` option